### DISCUSSION TITLE
pandas-ai error: "0.3 has type float, but expected one of: int"

### DISCUSSION THREAD

**Discussion by pmcray** [sinaptik-ai/pandas-ai#344] [Q&A]
I have what appears to me to be a perfectly vanilla dataframe, but I am getting the following error message. 

pandas_ai.run(df_first, prompt='Which endpoint has the highest quality score?')

Unfortunately, I was not able to answer your question, because of the following error:

0.3 has type float, but expected one of: int

Any thoughts?

**Comment by sandiemann** [comment]
@pmcray can you share your code snippet with the generate code output from llm
